portfolio/src/main/webapp/index.html

```diff
@@ -1,33 +1,101 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>Meagan Konst's Portfolio</h1>
-      <img src="/images/Selfie.jpg" alt="Meagan Selfie" width="300" height="400" class="center">
+
+<head>
+  <meta charset="UTF-8">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+<body>
+  <div id="content">
+  <h1>Meagan Konst's Portfolio</h1>
+  <img src="/images/Selfie.jpg" alt="Meagan Selfie" width="300" height="400" class="center">
```

Comment:
All of the `<img` tags should end with `/>` because they have no closing tag and are therefore unclosed currently.